RELEASE_NOTES.md

```diff
@@ -2,26 +2,26 @@
 
 [All Releases](https://github.com/masa-finance/masa-oracle/releases)
 
-## [0.0.9-beta](https://github.com/masa-finance/masa-oracle/releases) (2024)
+## [0.5.0](https://github.com/masa-finance/masa-oracle/releases) (2024)
```

Comment:
Can we please avoid to mix release notes changes among PRs?  Let's keep the scope of the PR tight to what we are adding/changing. 

This belongs to another PR or a follow-up to update the release notes.

Reply:
Makes sense. Will do this going forward.
The version was updated accordingly as specified above.